Mobile Buy SDK/Mobile Buy SDK Tests/BUYClient+CustomerTests.m

```diff
@@ -424,4 +458,12 @@ - (BUYAccountCredentials *)credentialsForFailure
 	return [BUYAccountCredentials credentialsWithItems:@[email, password]];
 }
 
+#pragma mark - Conveniences -
+
+- (NSString *)randomStringWithLength:(NSInteger)length
+{
+	NSString *UUID = [[NSUUID UUID] UUIDString];
```

Comment:
While every UUID is basically guaranteed to be unique, substrings are not. Does that affect your design?